### PR TITLE
Updated the geographical static data, ICs, SDFs and namelist settings in CCPP for p8c

### PR DESCRIPTION
**Pull request by ChunxiZhang-NOAA**
**Description**
To keep up with the CCPP physics p8c development, this PR updates the geographical static data, atm ICs for S2S, Suite Definition Files (SDFs) and namelist settings in CCPP (see issue #[737](https://github.com/NOAA-EMC/global-workflow/issues/737)).   The changes are:
1) Updated the files under fix/fix_fv3_fracoro directory
2) Pointed the atm ICs for S2S to /scratch2/NCEPDEV/climate/climpara/S2S/IC/GEFS-NoahMP-aerosols-p8c directory (on Hera)
(3) Deleted the RRTMGP related SDFs in build_ufs.sh
(4) Updated the namelist settings for the Thompson scheme and the NoahMP scheme.
(5) Adding to use new ufs.topo_edits_011818.nc file for MOM6 which when used with setting MOM6_ALLOW_LANDMASK_CHANGES=True for MOM6 at run time although this setting is turned off by default in the global-workflow.

For (1) and (5), I will work with global-workflow code managers to put them in the directories.

**Type of change**

- [X] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**
To do the tests on Hera:
(1) Clone the feature/p8c branch from https://github.com/ChunxiZhang-NOAA/global-workflow.git
(2) After running sh link_workflow.sh, cd ../fix, unlink fix_fv3_fracoro, ln -s /scratch1/NCEPDEV/hwrf/noscrub/Chunxi.Zhang/fix_fv3_fracoro .

- [X] Forecast-only test on Hera for both ATM only and S2SW coupled with aerosols.
  
**Checklist**

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
Removing dead CCPP suites from build_ufs.sh needs to be redone due to changes in develop, but otherwise looks good.

**Comment by WalterKolczynski-NOAA** [comment]
The new fix_fv3_fracoro also needs to be added to the canonical fix location instead of pointing to personal locations as well. Coordinate with @KateFriedman-NOAA

Are these additional files, or replacements? If they are replacements, are they backwards compatible?

**Comment by ChunxiZhang-NOAA** [comment]
> The new fix_fv3_fracoro also needs to be added to the canonical fix location instead of pointing to personal locations as well. Coordinate with @KateFriedman-NOAA
> 
> Are these additional files, or replacements? If they are replacements, are they backwards compatible?

Those are the replacements, and they are backwards compatible. I've tested those data with Prototype-P8b and it worked well.

**Comment by ChunxiZhang-NOAA** [comment]
> Removing dead CCPP suites from build_ufs.sh needs to be redone due to changes in develop, but otherwise looks good.

Redone.

**Comment by WalterKolczynski-NOAA** [comment]
Per #797, the UPP version needs to be updated as well to match that in UFS.

Otherwise, I'm waiting to hear from Kate that the new fix files are available.

**Comment by WalterKolczynski-NOAA** [comment]
Per #797, the UPP version needs to be updated as well to match that in UFS.

Otherwise, I'm waiting to hear from Kate that the new fix files are available.

**Comment by KateFriedman-NOAA** [comment]
@ChunxiZhang-NOAA What files in /scratch1/NCEPDEV/hwrf/noscrub/Chunxi.Zhang/fix_fv3_fracoro should I expect to see change if I rsync your copy into the main set? All of them? Only some? Please let me know if all or provide a list of the files. I need to know that I won't be erroneously changing files in the fix_fv3_fracoro folder that aren't meant to change but may be older in your copy (for example). Thanks!

**Comment by ChunxiZhang-NOAA** [comment]
@KateFriedman-NOAA  You can copy all of them or only the files changed. Under each C*.mx*_frac directory, the changed files are: 1) oro_*.tile?.nc files; 2) all files under subdirectory fix_sfc. The changed oro*.nc files are from /scratch2/NCEPDEV/ensemble/Bing.Fu/lake_p8_water_fraction2020 and the changed fix_sfc files are from /scratch2/NCEPDEV/ensemble/Bing.Fu/sfc_fixed_mar2022. I used those changed files to replace the files under /scratch1/NCEPDEV/global/glopara/fix_NEW/fix_fv3_fracoro to create /scratch1/NCEPDEV/hwrf/noscrub/Chunxi.Zhang/fix_fv3_fracoro

File: ush/parsing_namelists_FV3.sh [hunk 1]
```diff
@@ -344,6 +346,7 @@ cat >> input.nml <<EOF
   iopt_snf     = ${iopt_snf:-"4"}
   iopt_tbot    = ${iopt_tbot:-"2"}
   iopt_stc     = ${iopt_stc:-"1"}
+  iopt_trs     = ${iopt_trs:-"2"}
```
Comment (JessicaMeixner-NOAA) [inline ush/parsing_namelists_FV3.sh:349]: @ChunxiZhang-NOAA I saw that the following variables were added in ufs-weather-model: 

  do_RRTMGP          = @[DO_RRTMGP]
  active_gases       = 'h2o_co2_o3_n2o_ch4_o2'
  ngases             = 6
  lw_file_gas        = 'rrtmgp-data-lw-g128-210809.nc'
  lw_file_clouds     = 'rrtmgp-cloud-optics-coeffs-lw.nc'
  sw_file_gas        = 'rrtmgp-data-sw-g112-210809.nc'
  sw_file_clouds     = 'rrtmgp-cloud-optics-coeffs-sw.nc'
  rrtmgp_nGptsSW     = 112
  rrtmgp_nGptsLW     = 128
  rrtmgp_nBandsLW    = 16
  rrtmgp_nBandsSW    = 14
  doGP_cldoptics_LUT = @[DOGP_CLDOPTICS_LUT]
  doGP_lwscat        = @[DOGP_LWSCAT]

as well as: 

  lndp_type      = @[LNDP_TYPE]
  lndp_model_type = @[LNDP_MODEL_TYPE]
  LNDP_TAU=21600,
  LNDP_LSCALE=500000,
  ISEED_LNDP=2010,
  lndp_var_list = @[LNDP_VAR_LIST]
  lndp_prt_list = @[LNDP_PRT_LIST]
  
  
  Are any of these additional variables of consequence to the prototype runs?  I believe it's okay not to include either, but as this did show up as a difference between what we are running the ufs-waether-model regression tests and the workflow I wanted to double check.

**Comment by yangfanglin** [comment]
I'd recommend to keep this options even though they are not used by prototype p8 so that developers can use the workflow to experiment RRTMGP and land perturbation schemes.

**Comment by JessicaMeixner-NOAA** [comment]
> I'd recommend to keep this options even though they are not used by prototype p8 so that developers can use the workflow to experiment RRTMGP and land perturbation schemes.

I think these were additions, so they'd just need to be added here and they currently are not being added. 

Looking at diffs for the ufs-waether-model in between this version of ufs-weather-model and the previous tag, it also looks like the diag_table was not updated for pwatclm-> pwat I can either add that change in a subsequent PR or that can be added here?

File: ush/parsing_namelists_FV3.sh (hunk 1 above)
Comment (ChunxiZhang-NOAA) [reply ush/parsing_namelists_FV3.sh:349]: Those are not for the prototype runs. But as Fanglin suggested, I can include these in the workflow.

**Comment by KateFriedman-NOAA** [comment]
@ChunxiZhang-NOAA Thanks for those details and information on how you built your copy of `fix_fv3_fracoro`! I have pulled the updated `fix_fv3_fracoro` set from your Hera source into the `fix_NEW` set on Venus. I then synced the other `fix_NEW` sets on Mars, Hera, Jet, and Orion. I also made a new tarball copy of `fix_NEW` on HPSS:
/5year/NCEPDEV/emc-global/emc.glopara/fv3fix/fix_NEW_20220520.tar

**Comment by WalterKolczynski-NOAA** [comment]
> Per #797, the UPP version needs to be updated as well to match that in UFS.

Nevermind on this, I'm just removing the separate UPP clone in a separate PR.

**Comment by ChunxiZhang-NOAA** [comment]
> > I'd recommend to keep this options even though they are not used by prototype p8 so that developers can use the workflow to experiment RRTMGP and land perturbation schemes.
> 
> I think these were additions, so they'd just need to be added here and they currently are not being added.
> 
> Looking at diffs for the ufs-waether-model in between this version of ufs-weather-model and the previous tag, it also looks like the diag_table was not updated for pwatclm-> pwat I can either add that change in a subsequent PR or that can be added here?



> > I'd recommend to keep this options even though they are not used by prototype p8 so that developers can use the workflow to experiment RRTMGP and land perturbation schemes.
> 
> I think these were additions, so they'd just need to be added here and they currently are not being added.
> 
> Looking at diffs for the ufs-waether-model in between this version of ufs-weather-model and the previous tag, it also looks like the diag_table was not updated for pwatclm-> pwat I can either add that change in a subsequent PR or that can be added here?

I can make the changes pwatclm -> pwat in this PR.

File: ush/parsing_namelists_FV3.sh (hunk 1 above)
Comment (ChunxiZhang-NOAA) [reply ush/parsing_namelists_FV3.sh:349]: The namelist settings for land perturbation has been included already. It is controlled by switch $DO_LAND_PERT. By default $DO_LAND_PERT=.false and the namelist settings will not be generated.

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Conditionally approved pending additional testing

**Pull request merged**